.github/workflows/git-checks.yml

```diff
@@ -0,0 +1,12 @@
+name: Git Checks
+
+on: [pull_request]
+
+jobs:
+  block-fixup:
+    runs-on: ubuntu-18.04
```

Comment:
As the the doc: https://github.com/marketplace/actions/block-fixup-commit-merge

I hope they will soon migrate it to something newer :-)